### ISSUE TITLE
Password stored as plain text

### ISSUE THREAD

**Issue by yeSpud** [AtomicCoding/Quizlet-Bot#12]
When you give the program your password for your google account for auto-login, it stores the password as plaintext in the JSON file, which anyone can read

**Comment by YonatanAhituv** [comment]
@jeffrypig23 Agreed, I was thinking of doing encryption but I didn't really get to it. I'll do this when I get the chance to do a rewrite. Right now, it's not a full "release" and more of an alpha than anything else, the code is messy and it's badly written. But yeah, agreed.
